make/autoconf/jvm-features.m4

```diff
@@ -395,8 +395,14 @@ AC_DEFUN_ONCE([JVM_FEATURES_CHECK_ZGC],
         AC_MSG_RESULT([no, $OPENJDK_TARGET_OS-$OPENJDK_TARGET_CPU])
         AVAILABLE=false
       fi
-    elif test "x$OPENJDK_TARGET_OS-$OPENJDK_TARGET_CPU" = "xlinux-aarch64"; then
-      AC_MSG_RESULT([yes])
+    elif test "$OPENJDK_TARGET_CPU" = "aarch64"; then
```

Comment:
Should be "x$OPENJDK_TARGET_CPU" = "xaarch64"?

Reply:
added